### ISSUE TITLE
NoClassDefFoundError in the example using the SDK v2 and tracing (x-ray)

### ISSUE THREAD

**Issue by jeromevdl** [awslabs/aws-lambda-powertools-java#92]
Adding the sdk v2 in the pom of the example and using it in the App.java generate a NoClassDefFoundError at runtime.

**What were you trying to accomplish?**
Adding the SDK v2 and simply using it in the example generate this exception (see error logs below).

## Expected Behavior
Using the SDK v2 in the lambda function should not generate this kind of exception.

## Current Behavior
java.lang.NoClassDefFoundError "com/amazonaws/xray/handlers/config/AWSServiceHandlerManifest"

## Possible Solution
Problem with x-ray dependencies.

## Steps to Reproduce (for bugs)

1. Add the following dependencies (sdk v2) in the pom.xml: 

```
        <dependency>
            <groupId>software.amazon.awssdk</groupId>
            <artifactId>ssm</artifactId>
            <version>2.14.10</version>
        </dependency>
        <dependency>
            <groupId>software.amazon.awssdk</groupId>
            <artifactId>apache-client</artifactId>
            <version>2.14.10</version>
        </dependency>
```

2. Import the following classes in the App.java:

```
import software.amazon.awssdk.services.ssm.SsmClient;
import software.amazon.awssdk.services.ssm.model.GetParameterRequest;
```

3. Use the sdk v2 in the App.java within the `handleRequest` method:

```
        GetParameterRequest request = GetParameterRequest.builder()
                .name("/powertools-java/sample/simplekey")
                .build();
        SsmClient.create().getParameter(request);
```

## Environment

* **Powertools version used**: 0.2.0-beta
* **Packaging format (Layers, Maven/Gradle)**: maven
* **AWS Lambda function runtime:** java 8
* **Error logs**

```
  "errorMessage": "com/amazonaws/xray/handlers/config/AWSServiceHandlerManifest",
  "errorType": "java.lang.NoClassDefFoundError",
  "stackTrace": [
    "com.amazonaws.xray.interceptors.TracingInterceptor.initInterceptorManifest(TracingInterceptor.java:110)",
    "com.amazonaws.xray.interceptors.TracingInterceptor.<init>(TracingInterceptor.java:92)",
    "com.amazonaws.xray.interceptors.TracingInterceptor.<init>(TracingInterceptor.java:86)",
    "helloworld.App.handleRequest_aroundBody0(App.java:73)",
    "helloworld.App$AjcClosure1.run(App.java:1)",
    "org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:257)",
    "software.amazon.lambda.powertools.tracing.internal.LambdaTracingAspect.around(LambdaTracingAspect.java:52)",
    "helloworld.App.handleRequest_aroundBody2(App.java:65)",
    "helloworld.App$AjcClosure3.run(App.java:1)",
    "org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:257)",
    "software.amazon.lambda.powertools.logging.internal.LambdaLoggingAspect.around(LambdaLoggingAspect.java:94)",
    "helloworld.App.handleRequest(App.java:65)",
    "sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)",
    "sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)",
    "sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)",
    "java.lang.reflect.Method.invoke(Method.java:498)"
  ],
  "cause": {
    "errorMessage": "com.amazonaws.xray.handlers.config.AWSServiceHandlerManifest",
    "errorType": "java.lang.ClassNotFoundException",
    "stackTrace": [
      "java.net.URLClassLoader.findClass(URLClassLoader.java:382)",
      "java.lang.ClassLoader.loadClass(ClassLoader.java:424)",
      "java.lang.ClassLoader.loadClass(ClassLoader.java:357)",
      "com.amazonaws.xray.interceptors.TracingInterceptor.initInterceptorManifest(TracingInterceptor.java:110)",
      "com.amazonaws.xray.interceptors.TracingInterceptor.<init>(TracingInterceptor.java:92)",
      "com.amazonaws.xray.interceptors.TracingInterceptor.<init>(TracingInterceptor.java:86)",
      "helloworld.App.handleRequest_aroundBody0(App.java:73)",
      "helloworld.App$AjcClosure1.run(App.java:1)",
      "org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:257)",
      "software.amazon.lambda.powertools.tracing.internal.LambdaTracingAspect.around(LambdaTracingAspect.java:52)",
      "helloworld.App.handleRequest_aroundBody2(App.java:65)",
      "helloworld.App$AjcClosure3.run(App.java:1)",
      "org.aspectj.runtime.reflect.JoinPointImpl.proceed(JoinPointImpl.java:257)",
      "software.amazon.lambda.powertools.logging.internal.LambdaLoggingAspect.around(LambdaLoggingAspect.java:94)",
      "helloworld.App.handleRequest(App.java:65)",
      "sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)",
      "sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)",
      "sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)",
      "java.lang.reflect.Method.invoke(Method.java:498)"
    ]
  }
}
```

**Comment by msailes** [comment]
I have a feeling it's the inclusion of these dependencies in the tracing module. Testing it now.


```
        <dependency>
            <groupId>com.amazonaws</groupId>
            <artifactId>aws-xray-recorder-sdk-aws-sdk</artifactId>
        </dependency>
        <dependency>
            <groupId>com.amazonaws</groupId>
            <artifactId>aws-xray-recorder-sdk-aws-sdk-instrumentor</artifactId>
        </dependency>
```

**Comment by pankajagrawal16** [comment]
Closed as part of #90
